pages/partials/instructorGradingPanel.ejs

```diff
@@ -0,0 +1,37 @@
+<% if (authz_data.has_instructor_view || authz_data.has_course_permission_view) { %>
+    <div class="card mb-4 border-success">
+      <div class="card-header bg-success">Grading Panel</div>
+      <div class="card-body">
+        <% if (typeof submission_updated !== 'undefined' && submission_updated === true) { %>
+            <blockquote style="color: red">Submission updated</blockquote>
+        <% } %>
+        <form name="instance_question-manual-grade-update-form" method="POST">
+            <input type="hidden" name="__action" value="add_manual_grade">
+            <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+            <input type="hidden" name="assessment_instance" value="<%= assessment_instance.assessment_id %>">
+            <input type="hidden" name="assessment_question_id" value="<%= instance_question.assessment_question_id %>">
+            <div class="form-group">
+              <label for="submissionScore">Score:</label>
+              <div class="input-group">
+                <input type="text" class="form-control" id="submissionScore" name="submission_score" value="<%= score_perc %>" aria-describedby="scoreHelpBlock"/>  
```

Comment:
```suggestion
                <input type="number" class="form-control" id="submissionScore" name="submission_score" value="<%= score_perc %>" min="0" max="100" step="5" aria-describedby="scoreHelpBlock"/>  
```

I should point out that max is just where the slider stops if you keep pushing up. You can still allow values over 100.